packages/auth-core/src/pkce.ts

```diff
@@ -0,0 +1,14 @@
+import crypto from "node:crypto";
+
+export function createVerifierChallengePair(): {
+  verifier: string;
+  challenge: string;
+} {
+  const verifier = crypto.randomBytes(32).toString("hex");
```

Comment:
While this is strictly fine (hex is a subset of `base64url`), I think we should use `base64url` here. You probably copied this verbatim from my flawed first implementation, but I caught this discrepancy when writing the docs.

```suggestion
  const verifier = crypto.randomBytes(32).toString("base64url");
```